### DISCUSSION TITLE
是否有支持fec/red等冗余码机制的计划？

### DISCUSSION THREAD

**Discussion by alexliu-liu** [alibaba/xquic#125] [Ideas]
请问，是否有支持fec/red等冗余码机制的计划？

**Comment by ruiqizhou** [comment]
不在 Milestones 内的特性 代表短期内暂时没有计划。
可以通过 Issue 提交 New Feature Request，后续我们进行讨论，或者由社区进行贡献。

**Reply by alexliu-liu** [reply]
谢谢
